packages/orchestrator-ui-components/src/components/WfoForms/formFields/deprecated/ImsPortIdField.tsx

```diff
@@ -102,6 +102,9 @@ function ImsPortId({
                 return;
             }
 
+            // Prevents the value of the port dropdown remains
```

Comment:
This could be a worded a little clearer. 

```suggestion
            // Clears the value when another selection is made
```

Reply:
Yes, better!